build/yamls/base/crds.yml

```diff
@@ -187,6 +187,85 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  name: ippools.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha2
```

Comment:
why is this CRD starting with v1alpha2? and not v1alpha1?

Reply:
I figured ExternalIPPool and IPPool should be together, since they share IPRange. Do you think I should move IPRange and IPPool to v1alpha1?

Reply:
we can follow up on this one and see how we want to standardize such API introduction

Reply:
imho as ip ranges are attributes rather than standalone resources perhaps we should consider IPPool and ExternalIPPool two independent resources (and hence use v1alpha1 for IPPool)